presentation/matrix_view.py

```diff
@@ -246,19 +260,18 @@ def get_sla_level(val: MetricValue, warning_threshold: Threshold, error_threshol
             return SLALevel.WARNING
         return SLALevel.CRITICAL
 
-    def make_color_scale(self, z: List[SLALevelColumn]) -> List[Tuple[SLALevel, MatrixCellColor]]:
+    def make_color_scale(self) -> List[Tuple[SLALevel, MatrixCellColor]]:
```

Comment:
Why not make the "color_scale" instance variable set in the constructor? It does not depend on anything else than the configuration.